### ISSUE TITLE
BigInteger initialisation

### ISSUE THREAD

**Issue by createS0ftware** [komputing/KBigNumbers#3]
Hi 

What is the correct way to initialise BigInteger ?

I keep getting a NPE 

Thanks

**Comment by rmontagnin** [comment]
Could you please provide a snapshot of the code you are using? Also the platform you are developing for would be very useful

**Comment by ligi** [comment]
this seems related and has some details https://github.com/komputing/KHash/issues/11

**Comment by createS0ftware** [comment]
Yes, it is related. I thought the problem was in that library but it's in this one. 

In the Keccak class 

`  private val BIT_65 = BigInteger.ONE.shiftLeft(64)`

 ` private val MAX_64_BITS = BIT_65.subtract(BigInteger.ONE)`

BigInteger is not initialised prior to this call. I can't seem to find an appropriate way to initialise it

**Comment by rmontagnin** [comment]
@createS0ftware Are you using which module? The common one or the JVM one?

**Comment by createS0ftware** [comment]
JVM one

**Comment by rmontagnin** [comment]
I'm trying to reproduce the error but I don't seem able to do so. Could you please provide us with a full stack trace of the error?

**Comment by createS0ftware** [comment]
Ok - i will. However, i think i have found a possible fix. I am not importing the library correctly - the architecture of my kotlin multiplatform project, i think is wrong. 

Also i am targeting Android, IOS and JVM . I'll update yo tomorrow

Thanks
